addons/wowl/doc/services/hotkey.md

```diff
@@ -167,19 +162,15 @@ class MyComponent extends Component {
     this.hotkey = useService("hotkey");
   }
   mounted() {
-    this.hotkeyToken1 = this.hotkey.subscribe({
-      hotkey: "backspace",
-      callback: () => console.log("backspace has been pressed"),
-    });
-    this.hotkeyToken2 = this.hotkey.subscribe({
-      hotkey: "shift",
-      callback: () => console.log('Someone pressed on "shift"!'),
-    });
+    this.hotkeyToken1 = this.hotkey.registerHotkey("Backspace",
+      () => console.log('backspace has been pressed'));
+    this.hotkeyToken2 = this.hotkey.registerHotkey("Shift+P",
```

Comment:
is it really case-sensitive?

Reply:
nope, hotkey strings are case insensitive